source/règles/base.yaml

```
@@ -678,6 +678,30 @@
   références:
     Le régime des dirigeants: https://www.urssaf.fr/portail/home/employeur/creer/choisir-une-forme-juridique/le-statut-du-dirigeant/les-dirigeants-rattaches-au-regi.html
 
+- espace: contrat salarié
+  nom: stage
```

Pour éviter de poser la question
```suggestion
  nom: stage
  non applicable si: assimilé salarié
```